cli/azd/pkg/apphost/generate.go

```diff
@@ -378,6 +380,12 @@ func (b *infraGenerator) LoadManifest(m *Manifest) error {
 			b.connectionStrings[name] = *comp.ConnectionString
 		}
 
+		if comp.Type == "value.v0" && comp.Value != "" && !hasInputs(comp.Value) {
+			// a value.v0 resource with value not referencing inputs doesn't need any further processing
+			b.valueStrings[name] = comp.Value
+			continue
+		}
```

Comment:
Curios, does it make a difference to fold this short-circuit under the case for `value.v0` on L455?

Reply:
We can do that, sure.

I picked this place thinking it was a good idea to have it next to the `connectionString logic`, as it is somehow similar what we do for values.  But yes, it makes sense what you suggest. No difference